api/controllers/emailSubscribe.js

```diff
@@ -164,8 +161,8 @@ exports.unProtectedPut = async function (args, res, next) {
   var Project = mongoose.model('Project');
 
   // find the object ID based on the email address
-  await EmailSubscribe.findOne({ email: emailAddress }, null, function (err, entity) {
-    defaultLog.info('Err', err);
+  await EmailSubscribe.findOne({ _schemaName: 'EmailSubscribe', email: emailAddress }, null, function (err, entity) {
+    defaultLog.error('Err', err);
```

Comment:
not sure why there is a logger with error here, isn't at this point, still good, or am i missing something.

Reply:
This was originally coded as an "error-first callback"(https://nodejs.org/api/errors.html#error-first-callbacks) which used to be pretty conventional for async stuff in Node. Looks like this was half implemented though and there isn't proper error handling. I'll update this. Cheers